app/lib/actions/middleware/backend_services_check.rb

```diff
@@ -26,8 +26,8 @@ def plan(*args)
       protected
 
       def capsule_id(args)
```

Comment:
I found this method too difficult to understand for 2 lines of code. Since we're changing it- what do you think about this implementation?

```
def capsule_id(args)                                                                                                                                                         
  capsule_id = nil                                                                                                                                                           
  args.each do |arg|                                                                                                                                                         
    next unless arg.is_a? Hash                                                                                                                                               
                                                                                                                                                                             
    capsule_id = arg[:capsule_id] || arg[:smart_proxy_id]                                                                                                                    
    break if capsule_id                                                                                                                                                      
  end                                                                                                                                                                        
  capsule_id                                                                                                                                                                 
end
```

Reply:
Nice suggestion. Updated :)